prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/health-auth-care-types-page/health-auth-care-types-page.component.html

```diff
@@ -28,6 +28,7 @@
                   <app-options-form [form]="careType"
                                     fieldLabel="Care Type"
                                     controlName="careType"
+                                    optionLabel="careTypeName"
```

Comment:
On the initial load of the page you are not adapting the initial care types and they will not have this option label so nothing appears in the drop down.